changedetectionio/forms.py

```diff
@@ -318,10 +318,13 @@ def __call__(self, form, field):
                 if not self.allow_xpath:
                     raise ValidationError("XPath not permitted in this field!")
                 from lxml import etree, html
+                import elementpath
+                # xpath 2.0-3.1
+                from elementpath.xpath3 import XPath3Parser
```

Comment:
According to [installation-and-usage](https://elementpath.readthedocs.io/en/latest/introduction.html#installation-and-usage) of elementpath, `XPath3Parser` will support Xpath3.0 and 3.1. However, because of [SO answer](https://stackoverflow.com/a/62438044/20307768), it will support 2.0 too.